dockerfiles/cli/cli_tests.bats

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env bats
+# Copyright (c) 2012-2017 Red Hat, Inc
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# HOW TO RUN TEST?
+# To run tests, ensure you have:
+#     - installed bats
```

Comment:
Could we use a docker container to run the tests.
I would avoid to have bats as a requirement installed on the computer. (if it's there ppl can use it directly) but as all of dockerfiles are running with docker, I would expect to run tests within a docker container as well.

Also, as we have a build.sh script in /base folder, the script could have option to launch automatically the tests (with an option to skip the tests as well) so dev only launch a single command.